pom.xml

```diff
@@ -80,11 +80,11 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.2.5</version>
-           <configuration>
+           <!-- <configuration>
             <suiteXmlFiles>
                   <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
             </suiteXmlFiles>
-      </configuration>
+      </configuration> -->
```

Comment:
**Suggestion**: Un-comment the <configuration> section for the maven-surefire-plugin to ensure TestNG tests are executed as part of the build process. Commenting this out might skip the execution of tests. <br> **Code Suggestion**: 
 ```
 -<!-- <configuration>
 +<configuration>
 -</configuration> -->
 +</configuration>
 ```